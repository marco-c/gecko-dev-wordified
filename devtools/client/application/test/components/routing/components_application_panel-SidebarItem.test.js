/
*
Any
copyright
is
dedicated
to
the
Public
Domain
.
http
:
/
/
creativecommons
.
org
/
publicdomain
/
zero
/
1
.
0
/
*
/
"
use
strict
"
;
/
/
Import
libs
const
{
shallow
}
=
require
(
"
enzyme
"
)
;
const
{
createFactory
}
=
require
(
"
react
"
)
;
const
SidebarItem
=
createFactory
(
require
(
"
devtools
/
client
/
application
/
src
/
components
/
routing
/
SidebarItem
"
)
)
;
/
*
*
*
Test
for
SidebarItem
.
js
component
*
/
describe
(
"
SidebarItem
"
(
)
=
>
{
it
(
"
renders
the
expected
snapshot
with
manifest
page
"
(
)
=
>
{
const
wrapper
=
shallow
(
SidebarItem
(
{
page
:
"
manifest
"
}
)
)
;
expect
(
wrapper
)
.
toMatchSnapshot
(
)
;
}
)
;
it
(
"
renders
the
expected
snapshot
with
service
-
workers
page
"
(
)
=
>
{
const
wrapper
=
shallow
(
SidebarItem
(
{
page
:
"
service
-
workers
"
}
)
)
;
expect
(
wrapper
)
.
toMatchSnapshot
(
)
;
}
)
;
}
)
;
