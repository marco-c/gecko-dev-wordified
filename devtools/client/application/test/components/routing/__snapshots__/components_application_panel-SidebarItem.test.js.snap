/
/
Jest
Snapshot
v1
https
:
/
/
goo
.
gl
/
fbAQLP
exports
[
SidebarItem
renders
the
expected
snapshot
with
manifest
page
1
]
=
<
li
className
=
"
sidebar
-
item
"
>
<
Localized
attrs
=
{
Object
{
"
alt
"
:
true
"
title
"
:
true
}
}
id
=
"
sidebar
-
item
-
manifest
"
>
<
img
className
=
"
sidebar
-
item__icon
"
src
=
"
chrome
:
/
/
devtools
/
skin
/
images
/
application
-
manifest
.
svg
"
/
>
<
/
Localized
>
<
Localized
attrs
=
{
Object
{
"
title
"
:
true
}
}
id
=
"
sidebar
-
item
-
manifest
"
>
<
span
className
=
"
devtools
-
ellipsis
-
text
"
/
>
<
/
Localized
>
<
/
li
>
;
exports
[
SidebarItem
renders
the
expected
snapshot
with
service
-
workers
page
1
]
=
<
li
className
=
"
sidebar
-
item
"
>
<
Localized
attrs
=
{
Object
{
"
alt
"
:
true
"
title
"
:
true
}
}
id
=
"
sidebar
-
item
-
service
-
workers
"
>
<
img
className
=
"
sidebar
-
item__icon
"
src
=
"
chrome
:
/
/
devtools
/
skin
/
images
/
debugging
-
workers
.
svg
"
/
>
<
/
Localized
>
<
Localized
attrs
=
{
Object
{
"
title
"
:
true
}
}
id
=
"
sidebar
-
item
-
service
-
workers
"
>
<
span
className
=
"
devtools
-
ellipsis
-
text
"
/
>
<
/
Localized
>
<
/
li
>
;
